### ISSUE TITLE
Screenshots 

### ISSUE THREAD

**Issue by kittenvr** [Volcan4436/Neptune-Client#4]
Please show screenshots in readme

**Comment by OrangeMan-432** [comment]
Yeah, that would be really nice, I was just coming here to make an issue about that.

**Comment by Volcan4436** [comment]
DM me screenshots and I can make a seperate branch for them
